File: 20200416/code.py
money = int(input())
changeMoney = 1000 - money
arr = [500, 100, 50, 10, 5, 1]
count = 0

for i in arr:

  if changeMoney // i > 0:
    count += changeMoney // i
    changeMoney -=  i * (changeMoney // i)

print(count)

#https://www.acmicpc.net/problem/5585

File: 20200416/code.js
function solution(money) {
  chagneMoney = 1000 - money;
  arr = [500, 100, 50, 10, 5, 1];
  count = 0;

  arr.forEach(v => {
    let val = parseInt(chagneMoney / v, 10);
    if (val > 0) {
      count += val;
      chagneMoney -= v * val;
    }
  });
  return count;
}

console.log(solution(380));
